diff --git a/CMakeLists.txt b/CMakeLists.txt
index 51686b3..14f1a13 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,13 +1,6 @@
 cmake_minimum_required(VERSION 3.5)
 project(GameNetworkingSockets C CXX)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-
-include(FlagsMSVC)
-set(MSVC_RUNTIME "dynamic")
-configure_msvc_runtime()
-print_default_msvc_flags()
-
 option(Protobuf_USE_STATIC_LIBS "Build with a static Protobuf library" OFF)
 
 add_subdirectory(src)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 2276fba..ed54345 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -148,7 +148,6 @@ macro(gamenetworkingsockets_common GNS_TARGET)
 		if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
 			target_compile_definitions(${GNS_TARGET} PRIVATE
 				_CRT_SECURE_NO_WARNINGS
-				_ITERATOR_DEBUG_LEVEL=0
 			)
 			if(NOT Protobuf_USE_STATIC_LIBS)
 				target_compile_definitions(${GNS_TARGET} PRIVATE
diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 3efc4be..57bef95 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -1,7 +1,7 @@
 add_executable(
 	test_connection
 	test_connection.cpp)
-target_link_libraries(test_connection GameNetworkingSockets)
+target_link_libraries(test_connection GameNetworkingSockets ${CONAN_LIBS})
 
 #add_executable(
 #	test_flat
